frontend/src/locales/en.json

```diff
@@ -59,11 +59,12 @@
     "change": "change",
```

Comment:
"Email field in send GDD form must be called: Recipient" line 46 Receiver => Recipient

Reply:
ok .. super